frontend_tests/node_tests/drafts.js

```diff
@@ -89,6 +89,9 @@ test("draft_model add", ({override}) => {
     const ls = localstorage();
     assert.equal(ls.get("draft"), undefined);
 
+    const unread_count = $('<span class="unread_count"></span>');
+    $(".top_left_drafts").set_find_results(".unread_count", unread_count);
+
```

Comment:
I would consider doing this instead:

~~~ js
let expected_count = 0;

override(ui_util, update_unread_count_in_dom, (li, count) => {
  assert.equal(li, $(".top_left_drafts"));
  assert.equal(count, expected_count);
});

Reply:
You could also override `drafts.set_count` for many of these tests.  If you're not really testing the mechanism of how we set counts in a particular test, it's not really worth it to do all the zjquery stuff.

Reply:
(and then we should just have a direct test on `set_count`)

Reply:
The test for `set_count` would look like this:

~~~ js
 89 test("set_count", ({override}) => {
 90     override(ui_util, "update_unread_count_in_dom", (li, count) => {
 91         assert.equal(li, $(".top_left_drafts"));
 92         assert.equal(count, 99);
 93     });
 94
 95     drafts.set_count(99);
 96 });